app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/DeclarationJourneyActionProvider.scala

```diff
@@ -35,11 +36,13 @@ class DeclarationJourneyActionProvider @Inject()(defaultActionBuilder: DefaultAc
       override protected def refine[A](request: Request[A]): Future[Either[Result, DeclarationJourneyRequest[A]]] = {
```

Comment:
We should probably add the action builder to the skeleton journey controller methods also.

Reply:
I don't really see the value in that, over the next few days the Skeleton Journey will cease to exist, so it'll be a case of adding it, adding tests for it then removing them one by one as we churn out the full pages

Reply:
The value is:
1. that's what the ticket says so a QA will probably expect the behaviour.
2. Sarthak can test it now for all pages.
3. The functionality of all pages is consistent so is less confusing.

I'm happy to make the changes.